packages/manager/apps/container/src/container/nav-reshuffle/eloquant-survey/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { useEffect } from 'react';
+import { useShell } from '@/context';
+
+const EloquantSurvey = () => {
+  const shell = useShell();
+  const environment = shell.getPlugin('environment').getEnvironment();
+  const user = environment.getUser();
+  const region = environment.getRegion();
+  const languages = [
+    {
+      code: 'fr_FR',
+      short: 'fr',
+      long: 'french',
+    },
+    {
+      code: 'en_GB',
+      short: 'en',
+      long: 'english',
+    },
+    {
+      code: 'fr_CA',
+      short: 'fr',
+      long: 'french',
+    },
+    {
+      code: 'pl_PL',
+      short: 'pl',
+      long: 'polsky',
+    },
+    {
+      code: 'de_DE',
+      short: 'de',
+      long: 'DE',
+    },
+    {
+      code: 'es_ES',
+      short: 'es',
+      long: 'spanish',
+    },
+    {
+      code: 'it_IT',
+      short: 'it',
+      long: 'italian',
+    },
+    {
+      code: 'pt_PT',
+      short: 'pt',
+      long: 'portugues',
+    },
+  ];
+
+  const regions = {
+    EU: 'Europe',
+    CA: 'Asie',
+    US: 'USA'
+  };
+  const regionTerm = regions[region as 'EU'|'CA'|'US'];
+
+  useEffect(() => {
+    const { short, long } = languages.filter(
+      (lang) => lang.code === user.language,
+    )[0];
```

Comment:
```suggestion
    const { short, long } = languages.find(
      (lang) => lang.code === user.language,
    ) || {};
```
It is a bit more performant, also you can set default values if the `.find` return undefined